### DISCUSSION TITLE
How can I determine what setting is the "Use default setting" for notifications?

### DISCUSSION THREAD

**Discussion by myii** [WeblateOrg/weblate#5458] [Q&A]
For hosted weblate, I'm referring to this page:

* https://hosted.weblate.org/accounts/profile/#notifications

Many of the entries under `Managed projects` show `Use default setting` but I can't see what setting that refers to.  Perhaps it would be helpful to have the default setting in brackets, following the `Use default setting` option?  I.e. something like:

* `Use default setting (Daily digest)`

I didn't find the answer in the documentation, either:

* https://docs.weblate.org/en/latest/user/profile.html?highlight=notifications#notifications

**Comment by nijel** [comment]
The settings cascade and the tightest match has precedence. The "Use default setting" implies the cascading to upper level.

The precedence is:

1. Component
2. Project
3. Administered projects
4. Watched projects
5. Other projects

Suggestions how to clarify the description are welcome ;-).

**Reply by myii** [reply]
Unfortunately, I can't seem to get this right and keep missing when someone submits a translation update for my app (I don't receive an e-mail).  What's the best way to turn on _all_ notifications?  I'd rather have that for now than miss when someone makes the effort, which appears to be ignored by me.

For clarity, I've selected `Daily digest` for everything under `Translation notifications` for the component.  But I've also got _two_ identical looking components (which have slightly different URLs):

* Component: ut-tweak-tool/ut-tweak-tool -- https://hosted.weblate.org/accounts/profile/?notify_component=3784#notifications__3
* Component: ut-tweak-tool/ut-tweak-tool -- https://hosted.weblate.org/accounts/profile/?notify_component=3784#notifications__4

I'm now going to try to set the `Daily digest` on the second one as well.

---

Update: The second one won't save the `Daily digest` values and resets back to `Use default setting`.

**Reply by nijel** [reply]
The duplicate tabs should be fixed by 616ce8ff167fcbed5b0970ccc6ff132f152f0939.

The translated strings (and other notifications in this section) are sent only to your languages (as written on the top);

![image](https://user-images.githubusercontent.com/212189/119293805-79ff1600-bc53-11eb-806b-dfb4091402ed.png)

If you want to review each string, then using the review workflow might be a better choice than getting notified about each individual string, see https://docs.weblate.org/en/latest/workflows.html#dedicated-reviewers

**Reply by myii** [reply]
@nijel Thanks for fixing the duplicate tabs, I can confirm that's already applied for my project.

I'll check out the review workflow in the meantime and hope I can get things working right, so that I don't end up ignoring translators!  Thanks for the tip (and the fix).
